config/config_test.go

```diff
@@ -3150,3 +3178,44 @@ func TestTCF2FeatureOneVendorException(t *testing.T) {
 		assert.Equal(t, tt.wantIsVendorException, value, tt.description)
 	}
 }
+
+func TestAccountPriceFloors_validate(t *testing.T) {
+	type fields struct {
+		Enabled                bool
+		EnforceFloorRate       int
+		AdjustForBidAdjustment bool
+		EnforceDealFloors      bool
+		UseDynamicData         bool
+	}
```

Comment:
Can you delete the `fields` struct? It looks like you're planning to just specify the floors config in each test case as `pf` so I don't think you need this.

Reply:
removed `fields` struct?